krod/core/vector_store.py

```
@@ -8,9 +8,11 @@
 from uuid import uuid4
 # import numpy as np
 from qdrant_client import QdrantClient
-from qdrant_client.http.models import Distance, VectorParams, PointStruct, Filter, FieldCondition, MatchValue
+from qdrant_client.http.models import Distance, VectorParams, PointStruct, Filter, FieldCondition, MatchValue, CollectionStatus
```

Flake8 (7.2.0)

[error] 'qdrant_client.http.models.CollectionStatus' imported but unused

(F401)